iOS/Data/Model/Course+showDates.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import Common
+import Foundation
+
+extension Course {
+
+    public func showCourseDates(viewController: UIViewController) {
```

Comment:
Similar to the comment above, this extension is tightly coupled to the view hierarchy. To resolve this, I suggest creating an extension on `CourseDateListViewController` instead and not calling `.present` in the extension.